src/test/java/com/wikia/webdriver/testcases/mobile/curatedcontenttests/EditorTests.java

```diff
@@ -33,22 +33,25 @@
 
   private static String FEATURED_CONTENT_SELECTOR = ".featured-content";
   private static String CURATED_CONTENT_SELECTOR = ".curated-content";
+  private static Navigate navigate;
 
   @BeforeMethod(alwaysRun = true)
   public void beforeMethod() {
     new CuratedContent().clear();
+    navigate = new Navigate();
   }
 
   @Test(groups = "MercuryCuratedEditorTest_001")
   @RelatedIssue(issueID = "XF-241")
   public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
-    Boolean result = new ArticlePage().open(MobileSubpages.ECC_MAIN_PAGE).getCuratedMainPage()
-        .isCuratedElementVisible(FEATURED_CONTENT_SELECTOR);
+
+    navigate.toPageByPath(MobileSubpages.ECC_MAIN_PAGE);
+    CuratedMainPagePageObject curatedMainPage = new ArticlePage().getCuratedMainPage();
+    Boolean result = curatedMainPage.isCuratedElementVisible(FEATURED_CONTENT_SELECTOR);
 
     Log.info(String.format("Curated content is visible: %s", result));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Curated content is visible: %s" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)